addons/sale/views/sale_order_views.xml

```diff
@@ -712,6 +712,7 @@
                                 </div>
                                 <field name="partner_invoice_id" groups="!account.group_delivery_invoice_address" invisible="1"/>
                                 <field name="analytic_account_id" context="{'default_partner_id':partner_invoice_id, 'default_name':name}" readonly="invoice_count != 0 and state == 'sale'" groups="analytic.group_analytic_accounting" force_save="1"/>
+                                <field name="journal_id" groups="base.group_no_one" readonly="invoice_count != 0 and state == 'sale'"/>
```

Comment:
Why is it only in debug mode ? It looks like it's useful for a sales manager for instance right ?

Reply:
(was part of plan) @dalaOdoo Any opinion ?

Reply:
no reason